### ISSUE TITLE
convert x/y coords to lat / lon for extracts

### ISSUE THREAD

**Issue by camallen** [zooniverse/aggregation-for-caesar#106]
My use case is the PRN where I need to provide the raw extracted data for each task type with the marking task data converted to lat / lon for each subject.

Currently i'm taking the code here and modifying it for use for generic workflows https://github.com/AroneyS/prn_data_extract/blob/master/cleanup_workflow_output.py which was heavily inspired by the work done in https://github.com/vrooje/Data-digging/blob/1eece78c0f4dfc6b700e3f631d37681b6a8b7bf6/example_scripts/planetary_response_network/caribbean_irma_2017/extract_markings_to1file.py#L197

It would be ideal to have a robust solution for converting known point data on geolocated subjects to lat / lon coords. Is this feasible at all?

**Comment by CKrawczyk** [comment]
For this to work the subject metadata would have to be passed along with the classification information.  This can be done with Caesar's built in `PluckFieldExtractor` and `FirstExtractReducer`.  The output of this reducer would have to be passed into a new external reducer to convert the points to lat/lon, this can't be done yet (but I think is currently being worked on for TESS, the same system would be used for calculating user weighted reductions).

**Comment by camallen** [comment]
Yes, i'm supplying subject metadata (subjects export file) as an input to my [converter script](https://github.com/zooniverse/tprn-data-processing/blob/27d698782a245030000a81729e7d794648d46916/data_conversion/convert_to_ibcc.py#L120). 

My use case is completely offline right now and only for extracts as the consuming client process wants the raw data only. The above workflow of reducer converting the points doesn't quite fit, though long term it looks the most likely place to convert the point data to lat/lon.
